Simplenote/src/main/java/com/automattic/simplenote/Simplenote.java

```diff
@@ -72,7 +73,11 @@ public void onCreate() {
         super.onCreate();
 
         CrashUtils.initWithContext(this);
-        AppLockManager.getInstance().enableDefaultAppLockIfAvailable(this);
+        if (DefaultAppLock.isSupportedApi()) {
+            SimplenoteAppLock appLock = new SimplenoteAppLock(this);
+            AppLockManager.getInstance().setCurrentAppLock(appLock);
+            appLock.enable();
+        }
```

Comment:
```suggestion
        SimplenoteAppLock appLock = new SimplenoteAppLock(this);
        AppLockManager.getInstance().setCurrentAppLock(appLock);
        appLock.enable();
```

Since the `minSdkVersion` is `23` and `DefaultAppLock.isSupportedApi()` is `Build.VERSION.SDK_INT >= Build.VERSION_CODES.ICE_CREAM_SANDWICH`, the `if` condition is always `true`, right?  If so, we can remove it.

Reply:
Good catch, done d2a255c